monitor.cc

```diff
@@ -0,0 +1,133 @@
+#ifndef CPP_CLIENT_LINE_H_
+#define CPP_CLIENT_LINE_H_
+#include <iostream>
+#include <string>
+#include <grpcpp/grpcpp.h>
+#include <gflags/gflags.h>
+
+#include "chirpsl.grpc.pb.h"
+#include "chirpsl.pb.h"
+#include "sl_client_grpc.h"
+
+DEFINE_string(options, "register,user,chirp,reply,follow,read,monitor",
+                 "comma-separated list of options a user may execute in 'chirp' menu");
+
+DEFINE_string(register, "", "the username you wish to register with");
+DEFINE_string(user, "", "the username you wish to login with");
+DEFINE_string(chirp, "", "the text you wish to chirp");
+DEFINE_string(reply, "", "the id of the chirp you wish to reply to");
+DEFINE_string(follow, "", "the username you wish to follow");
+DEFINE_string(read, "", "the id of the chirp you wish to read");
+DEFINE_bool(monitor, false, "call this to monitor chirps by people you follow");
+
+int main(int argc, char **argv)
+{
+	// gflags::ParseCommandLineFlags(&argc, &argv, true);
```

Comment:
Remove commented out lines. I think this file was for testing, you should probably remove it from your repo